src/chatClient.cpp

```diff
@@ -393,6 +393,42 @@ int Client::importMessages(const char *externalDbPath)
         chatd::Chat &chat = chatroom->chat();
         karere::Id chatid = chatroom->chatid();
 
+        // Import last seen from external db
+        Id lastSeenId;
+        SqliteStmt stmtLastSeen(dbExternal, "select last_seen from chats where chatid=?");
+        stmtLastSeen << chatid;
+        stmtLastSeen.stepMustHaveData();
+        lastSeenId = stmtLastSeen.uint64Col(0);
+        chat.setLastSeenId(lastSeenId);
+
+        SqliteStmt stmtLastSeenIdx(dbExternal, "select idx from history where chatid = ? and msgid = ?");
+        stmtLastSeenIdx << chatid << lastSeenId;
+        if (stmtLastSeenIdx.step())
+        {
+            chat.setLastSeenIdx(stmtLastSeenIdx.intCol(0));
+        }
+        chatroom->chat().handleLastImportedSeen(lastSeenId);
+
+        // Import last received from external db
+        Id lastRecvId;
+        SqliteStmt stmtRecv(dbExternal, "select last_recv from chats where chatid=?");
```

Comment:
No clients are using/updating the `RECEIVED`, so probably we can simply skip this part.
I'll check with the team if there are plans to restore the feature in a future. In that case, better to keep it.